planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -2538,4 +2538,115 @@ double calcLaneChangeBuffer(
   return num_lane_change * calcTotalLaneChangeDistanceWithBuffer(common_param) +
          length_to_intersection;
 }
+
+bool applyVelocitySmoothing(
```

Comment:
It is not recommended to use the smoother here. Currently, smoother is used in a few nodes, but we do not want to add more from a code management point. Now we can't change the role of the smoother easily due to the dependencies, which is the drawback.
I assume the reason for using the smoother is to accurately predict the future velocity, but there is a limitation of the hierarchical architecture. We should solve this issue in the architecture design.

For now, in the avoidance module, the future ego velocity can be assumed as a constant, or increasing with a constant acceleration up to the velocity limit.

Reply:
OK. I'll remove smoother.

Reply:
Removed smoother.